orga/translations/en.json

```
@@ -1,15 +1,56 @@
 {
   "current-lang": "en",
+  "common": {
+    "form": {
+      "answer": {
+        "no": "No",
+        "yes": "Yes"
+      },
+      "action": {
+        "cancel": "Cancel"
+      }
+    }
+  },
   "navigation": {
-    "main" : {
-      "campaigns" : "Campaigns",
+    "main": {
+      "campaigns": "Campaigns",
       "documentation": "Documentation",
-      "students-sco" :"Students",
-      "students-sup" :"Students",
-      "team" : "Team"
+      "students-sco":"Students",
+      "students-sup":"Students",
+      "team": "Team"
     },
     "user-logged-menu": {
       "logout": "Log out"
     }
+  },
+  "pages": {
+    "create-campaign": {
+      "main-title": "Create a campaign",
+      "form": {
+        "action": {
+          "submit": "Create a campaign"
+        },
+        "answer": {
+          "assessment": "Assess participants",
+          "profiles-collection": "Collect the participants Pix profiles"
+        },
+        "label": {
+          "external-id": "Do you need an external user ID ?",
```

pas d'espace avant le point d'interrogation en anglais
```suggestion
          "external-id": "Do you need an external user ID?",
```
(idem pour les autres avant point d'interrogation)